engine/src/main/scala/cromwell/engine/workflow/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -0,0 +1,165 @@
+package cromwell.engine.workflow
+
+import java.nio.file.Paths
+
+import akka.actor.{Actor, Props}
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.LazyLogging
+import cromwell.engine._
+import cromwell.engine.backend.runtimeattributes.CromwellRuntimeAttributes
+import cromwell.engine.backend.{Backend, BackendType, CromwellBackend}
+import cromwell.engine.workflow.MaterializeWorkflowDescriptorActor.{MaterializationFailure, MaterializationSuccess}
+import cromwell.util.TryUtil
+import spray.json._
+import wdl4s._
+
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+import scalaz.Scalaz._
+import scalaz.Validation.FlatMap._
+
+object MaterializeWorkflowDescriptorActor {
```

Comment:
I'm sorry but what is the final conclusion here? @Horneth 's final name contradicts the way that we decided to name Actors (by appending the term 'Actor' to the classes)